{
  throw new Fault(new IOException("It's an IOException."));
}
